### ISSUE TITLE
[Suggestion] Choose Void Dimensions

### ISSUE THREAD

**Issue by EmosewaGamer** [APengu/IslandCraft#1]
Could you please add a config option for selecting which dimensions should be voids, or at least selecting whether the Nether should be a void or not.

**Comment by APengu** [comment]
Sure, thanks for the suggestion

**Comment by EmosewaGamer** [comment]
Yeah, being able to choose voided dimensions would be better as then you could have a regular or custom world in which people spawn in structures.
